examples/formatter-golangci-lint.toml

```diff
@@ -0,0 +1,6 @@
+# Example generated by ../examples.sh
+[formatter.golangci-lint]
+command = "golangci-lint-run"
+excludes = []
+includes = ["*.go"]
+options = []
```

Comment:
I just noticed that the example is not super useful because it won't work outside of Nix